src/components/BookForm.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
```

Comment:
**REQUIRED**
Let's separate container and presentation components. 
Since `BookForm` has access to the state, let's place it in the `src/container` directory.
See this [link](https://scotch.io/courses/5-essential-react-concepts-to-know-before-learning-redux/presentational-and-container-component-pattern-in-react) for more information.